### ISSUE TITLE
Radio Buttons are not switchable by tabs

### ISSUE THREAD

**Issue by heyAfsar** [heyAfsar/RegisterationFormHTML#19]
The user should be able to navigate across form using tabs. 
The application is not aligned with power user's requirements.
